src/shared/components/case-list-filters/case-list-filters.component.ts

```diff
@@ -0,0 +1,241 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { ActivatedRoute, ActivatedRouteSnapshot } from '@angular/router';
+import { FormGroup } from '@angular/forms';
+import 'rxjs/add/operator/do';
+import { Jurisdiction, CaseState, CaseTypeLite, WorkbasketInputModel } from '../../domain';
+import { READ_ACCESS } from '../../domain/case-view/access-types.model';
+import { JurisdictionService, AlertService, WindowService, OrderService, WorkbasketInputFilterService,
+  DefinitionsService } from '../../services';
+
+const FORM_GROUP_VAL_LOC_STORAGE = 'case-list-filters-form-group-value';
+const SAVED_QUERY_PARAM_LOC_STORAGE = 'savedQueryParams';
+@Component({
+  selector: 'ccd-case-list-filters',
+  templateUrl: './case-list-filters.component.html',
+  styleUrls: ['./case-list-filters.component.scss']
+})
+export class CaseListFiltersComponent implements OnInit {
```

Comment:
The logic here should be much simpler without duplicating the workbasket filters component but just wrapping it similar to search filters and search filters wrapper component.

Reply:
Done